src/themes/default/components/core/ProductGalleryZoom.vue

```diff
@@ -50,8 +50,8 @@ export default {
   },
   components: {
     'no-ssr': NoSSR,
-    Carousel,
-    Slide
+    'Carousel': () => import('vue-carousel').then(Slider => Slider.Carousel),
```

Comment:
we should discuss this
because it is showed on initial view. There is no v-if to delay it's load so i think it should stay statically loaded

Reply:
but the load is delayed to the point when initial js is parsed and executed so the preformance gain is still valid